people/time-off.md

```diff
@@ -44,6 +44,25 @@ Follow the process described above in this case - you are welcome to describe yo
 In this case, our guidelines around letting others know ahead of time are somewhat relaxed, as sick leave may happen unexpectedly.
 Just do the best you can, and try to balance your own constraints with respect for the team’s bandwidth and planning.
 
+(time-off:annual-expected)=
+## End-of-year expected time off
+
+The **final 14 days of the year are "expected time off" for all of 2i2c**.
```

Comment:
Usually, people do not start until the 3rd of Jan (maybe more?), maybe we should include those as well, so people are not expecting us to start working on Jan 1st?

Reply:
How about `**All of 2i2c has "expected time off" from December 17th through January 3rd of the following year.**`

Reply:
I would try to avoid specific dates because this will fluctuate each year, so maybe?:

`final 14 days and first 3 days of the year(s)`